frontend/src/metabase/querying/components/FilterValuePicker/FilterValuePicker.tsx

```diff
@@ -78,9 +85,9 @@ function FilterValuePicker({
   }
 
   return (
-    <SelectValuePicker
-      data={data}
-      value={value}
+    <StaticValuePicker
+      fieldValues={fieldData?.values ?? []}
```

Comment:
We can come here with field values if `has_more_values` is `true` but the field is not searchable, e.g. there is no `searchFieldId`. In this case there is nothing we can do but to display the list of values we have and allow to create new items.